tests/unit/viahtml/stats_test.py

```diff
@@ -0,0 +1,147 @@
+        expected = [
+            ("Custom/Alive/Metrics", 1),
+            ("Custom/Alive/App", 1),
+            ("Custom/System/Load", 16),
+            ("Custom/ListenQueue/Size", 12),
+            ("Custom/ListenQueue/Errors", 13),
+            ("Custom/SignalQueue/Size", 14),
+            (
+                "Custom/Socket/Queue/Size",
+                {"total": 17, "count": 1, "min": 17, "max": 17, "sum_of_squares": 289},
+            ),
+            (
+                "Custom/Socket/Queue/Max",
+                {
+                    "total": 100,
+                    "count": 1,
+                    "min": 100,
+                    "max": 100,
+                    "sum_of_squares": 10000,
+                },
+            ),
+            ("Custom/Worker/Count/Accepting", 5),
+            ("Custom/Worker/Count/Max", 7),
+            (
+                "Custom/Worker/Request/Total",
+                {"total": 14, "count": 5, "min": 0, "max": 12, "sum_of_squares": 148},
+            ),
+            (
+                "Custom/Worker/Request/Failed",
+                {"total": 7, "count": 5, "min": 0, "max": 5, "sum_of_squares": 29},
+            ),
+            (
+                "Custom/Worker/Killed",
+                {"total": 7, "count": 5, "min": 0, "max": 4, "sum_of_squares": 21},
+            ),
+            (
+                "Custom/Worker/Memory/Resident[kB]",
+                {
+                    "total": 44796.48,
+                    "count": 5,
+                    "min": 3.12,
+                    "max": 44784.0,
+                    "sum_of_squares": 2005606694,
+                },
+            ),
+            (
+                "Custom/Worker/AverageResponseTime[ms]",
+                {
+                    "total": 177.04,
+                    "count": 2,
+                    "min": 0.34,
+                    "max": 176.7,
+                    "sum_of_squares": 31223,
+                },
+            ),
+            (
+                "Custom/Worker/UpTime[ms]",
+                {
+                    "total": 354.56,
+                    "count": 1,
+                    "min": 354.56,
+                    "max": 354.56,
+                    "sum_of_squares": 125712,
+                },
+            ),
+            (
+                "Custom/Worker/Memory/Virtual[kB]",
+                {
+                    "total": 748300.0,
+                    "count": 1,
+                    "min": 748300.0,
+                    "max": 748300.0,
+                    "sum_of_squares": 559952890000,
+                },
+            ),
+        ]
+
+        for metric_name, value in expected:
+            assert (metric_name, value) in results
+
+    @pytest.fixture
+    def stats(self):
+        return UWSGINewRelicStatsGenerator(stats_endpoint=STATS_ENDPOINT)
+
+    @pytest.fixture
+    def json_response(self):
+        with open(resource_filename("tests", "unit/viahtml/stats_test.json")) as handle:
+            return json.load(handle)
```

Comment:
I'm not a fan of putting fixtures in separate JSON files. I think you should just inline the dict into the Python file where it's used. Put it down at the bottom of the file so it's out of the way. Otherwise the reader needs to maintain a mapping in their head between this `json_response` fixture and the `stats_test.json` file (different name) that it came from